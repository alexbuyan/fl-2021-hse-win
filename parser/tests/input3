4 \
1 - 0 \
2 - 3 \
3 - 3 \
4 - 2 \
1 -> 2 : 1...9 \
1 -> 3 : 0 \
2 -> 2 : 0...9 \
3 -> 4 : 0...9 \
4 -> 4 : 0...9 \