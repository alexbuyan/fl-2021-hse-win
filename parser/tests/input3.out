Alphabet:
	{0, 1, 2, 3, 4, 5, 6, 7, 8, 9}
States:
	(name: 1, type: start)
	(name: 2, type: terminal)
	(name: 3, type: terminal)
	(name: 4, type: stock)
Edges:
	(from: 1, to: 2, moves: 1,2,3,4,5,6,7,8,9)
	(from: 1, to: 3, moves: 0)
	(from: 2, to: 2, moves: 0,1,2,3,4,5,6,7,8,9)
	(from: 3, to: 4, moves: 0,1,2,3,4,5,6,7,8,9)
	(from: 4, to: 4, moves: 0,1,2,3,4,5,6,7,8,9)
